crates/parser/src/ast.rs

```diff
@@ -18,7 +18,7 @@ pub enum ModuleStmt {
     Use(Node<Use>),
     TypeAlias(Node<TypeAlias>),
     Contract(Node<Contract>),
-    Constant(Node<ConstantDecl>),
+    Constant(Box<Node<ConstantDecl>>),
```

Comment:
clippy was complaining about this variant being ~2x the size of the next largest